src/main/scala/com/fulcrumgenomics/umi/ConsensusCallingIterator.scala

```diff
@@ -67,10 +67,11 @@ class ConsensusCallingIterator[ConsensusRead <: SimpleRead](sourceIterator: Iter
       groupingIterator.flatMap(caller.consensusReadsFromSamRecords)
     }
     else {
-      ParIterator(groupingIterator, threads=threads).flatMap { rs =>
+      ParIterator(groupingIterator, threads=threads, chunkSize=chunkSize).map { rs =>
         val caller = callers.get()
         caller.synchronized { caller.consensusReadsFromSamRecords(rs) }
-      }.toAsync(chunkSize * 8)
+      }.toAsync(chunkSize).flatten
```

Comment:
I am confused with the chunk size passed to `toAsync` is reduced.  Don't we want this value to multiples of the input `chunkSize`.  Furthermore, since the `ParIterator` is presumably processing `threads` chunks at any one time, I think this should be `chunkSize * threads`?
 
```suggestion
      }.toAsync(chunkSize * threads).flatten
```